src/components/Table/TableToolbar/TableToolbar.jsx

```diff
@@ -71,7 +71,7 @@ const StyledTableBatchActions = styled(TableBatchActions)`
 `;
 
 const StyledTooltipContainer = styled.div`
-  padding: 1rem 1rem 0 0;
+  padding: 1rem 0;
```

Comment:
could this particular styled component be refactored into the table-toolbar sass file since we are touching both files anyways?

Reply:
Yep definitely. I'll get this changed shortly

Reply:
@davidicus updated!